crates/routes/src/feeds.rs

```diff
@@ -173,6 +198,28 @@ async fn get_feed(
   )
 }
 
+fn get_limit_and_page(req: &HttpRequest) -> (i64, i64) {
+  let mut limit: i64 = RSS_FETCH_LIMIT;
+  let mut page: i64 = 1;
+
+  req.query_string().split('&').for_each(|p| {
+    let mut param = p.split('=');
+    let key = param.next().unwrap_or("");
+    let value = param.next().unwrap_or("");
+    match key {
+      "limit" => {
+        limit = value.parse().unwrap_or(limit);
+      }
+      "page" => {
+        page = value.parse().unwrap_or(page);
+      }
+      _ => {}
+    }
+  });
```

Comment:
This is not how you should read params.

https://actix.rs/docs/extractors#query

Reply:
Oof yea, totally missed that.